src/main/java/com/gooddata/warehouse/WarehouseS3Credentials.java

```diff
@@ -101,13 +109,6 @@ public DateTime getUpdated() {
     }
 
     /**
-     * @return URI of the user who updated the credentials last
-     */
-    public String getUpdatedBy() {
```

Comment:
How will you access `updatedBy` in the converter in `gdc-datawarehouse` if you delete the getter?